pkg/restapi/operation/respond.go

```diff
@@ -275,3 +275,18 @@ func writeUpdateDocumentFailure(rw http.ResponseWriter, errUpdateDoc error, docI
 		logger.Errorf(messages.UpdateDocumentFailure+messages.FailWriteResponse, docID, vaultID, errUpdateDoc, errWrite)
 	}
 }
+
+func writeDeleteDocumentFailure(rw http.ResponseWriter, errDeleteDoc error, docID, vaultID string) {
+	logger.Infof(messages.DeleteDocumentFailure, docID, vaultID, errDeleteDoc)
+
+	if errDeleteDoc == messages.ErrDocumentNotFound || errDeleteDoc == messages.ErrVaultNotFound {
```

Comment:
Is VaultNotFound considered 'bad request' or 'not found'?

Reply:
I considered it as 'not found' since we do it this way in the readDocumentHandler

Reply:
sure, np